agent/api/ecsclient/client_unspecified.go

```diff
@@ -13,11 +13,11 @@
 // express or implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
-package app
+package ecsclient
```

Comment:
Dunno if this is github or git being weird, but this isn't a file move as much as a deletion of cruft combined with addition of a new function. Would be nice if the git history looked like that too.